cartridges/int_affirm_controllers/cartridge/scripts/payment/instrument/BASIC_CREDIT.js

```diff
@@ -28,7 +28,7 @@ exports.add = function (basket) {
     var cardNumber = parametersMap.number.stringValue;
     var cardHolder = parametersMap.cardholder_name.stringValue;
     var cardSecurityCode = parametersMap.cvv.stringValue;
-    var cardType = 'Master Card';
+    var cardType = 'Master';
```

Comment:
@taehyunlim should this be Visa?

Reply:
Good catch. That change was checked in to the master branch but I will make sure this gets updated.